docs/guides/frontity-query-options.md

```diff
@@ -0,0 +1,33 @@
+# Frontity Query Options
+
+"Frontity Query Options" are query string parameters added to the URL that start with `frontity_`.
+
+For example:
+
+```
+https://example.com/my_post?frontity_name=Site-name
+```
+
+Any query parameter in the URL that starts with `frontity_` is a "Frontity Query Option".
+
+They can be used to dynamically configure certain options in the Frontity state.
+
+> It is important to note that Frontity Query Options do not form part of the canonical link. This means that whatever is passed in the `frontity_` query parameter will not be matched by handlers or used to fetch data from WordPress. Its purpose is merely to allow dynamic configuration of Frontity.
+
+Query string parameters that start with `frontity_` are reserved to **send special information to Frontity**, i.e. they are not related to the URL that needs to be rendered (e.g. a post or a page), but instead are used to change the Frontity configuration dynamically.
```

Comment:
Do you think we could somehow merge those 3 paragraphs into two? I ask because we write in first paragraph: 

> can be used to dynamically configure certain options

and then in the third: 

> but instead are used to change the Frontity configuration dynamically.

Reply:
TBH I'm not sure what you're suggesting here.

Reply:
I'm sorry, I think I wasn't clear which paragraphs I'm referring to. I meant that there is a bit of repetition here. We mention in two places that the purpose of Frontity Options is to allow "dynamic configuration".

However... now that I'm reading this again, I think that a bit of redundancy actually makes it more clear to the user so please disregard the previous comment! 🙂